calledudeBot/Common/Hooky.cs

```diff
@@ -19,7 +19,7 @@ public sealed class Hooky
         private int position;
         private readonly TwitchBot twitchBot;
 
-        public Hooky(TwitchBot twitchBot) => this.twitchBot = twitchBot;
+        public Hooky(ref TwitchBot twitchBot) => this.twitchBot = twitchBot;
```

Comment:
Why the `ref`?

Reply:
A remnant from when I was trying something out 👍